dds_web/static/swagger.yaml

```diff
@@ -466,7 +466,48 @@ paths:
                     type: string
       requestBody:
         $ref: "#/components/requestBodies/Token"
-
+  ### S3 OPERATIONS #########
+  /s3/proj:
+    get:
+      tags:
+        - s3
+      summary: Get the project S3 keys
+      operationId: retrieveUserInfo
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+        - $ref: "#/components/parameters/projectID"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  bucket:
+                    type: string
+                    example: bucketname
+                  keys:
+                    type: object
+                    properties:
+                      access_key:
+                        type: string
+                        example: access_key
+                      secret_key:
+                        type: string
+                        example: secret_key
+                  url:
+                    type: string
+                    example: url
+                  safespring_project:
+                    type: string
+                    example: sfsp_proj
```

Comment:
reorder so that it matches the order in s3.py:

```
        return {
            "safespring_project": sfsp_proj,
            "url": url,
            "keys": keys,
            "bucket": bucketname,
        }
```
again, for consistancy only